cvat-ui/src/base.scss

```diff
@@ -21,7 +21,8 @@ $danger-icon-color: #ff4136;
 $info-icon-color: #0074d9;
 $objects-bar-tabs-color: #bebebe;
 $objects-bar-icons-color: #242424; // #6e6e6e
-$active-object-item-background-color: #d8ecff;
+$active-label-background-color: #d8ecff;
+$object-item-border-color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5722655)